lib/smart_answer/erb_renderer/question_options_helper.rb

```diff
@@ -0,0 +1,11 @@
+module SmartAnswer
+  module ErbRenderer::QuestionOptionsHelper
+    def options(options = nil)
+      if options
+        @options = options
+      else
+        @options || {}
+      end
+    end
```

Comment:
I know this is something you didn't introduce, but this makes makes my eyes pop a bit for the ambiguity of this method.

Reply:
Yup - can look at fixing in another PR